doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -150,3 +149,45 @@ WARNING: You should only import Dashbuilder data to a clean installation of {PRO
 . Click *Import*.
 
 endif::DM[]
+
+ifdef::PAM[]
+
+== Deploying dashboards from {CENTRAL} on Dashbuilder Runtime
+
+You can automatically deploy the dashboards from {CENTRAl} on Dashbuilder Runtime. {CENTRAl} is linked to Dashbuilder Runtime using a gradual export feature.
+
+.Prerequisites
+
+* Dashbuilder Runtime is configured on your system.
+* You have set the `dashbuilder.runtime.multi` system property to `true` in the `standalone.xml` file.
+* You have set the value of `dashbuilder.runtime.location` system property to the Dashbuilder Runtime URL. For example:
```

Comment:
Per IBM SG, the preference is to have a complete sentence before colons, to avoid things like "for example", "the following:", "as follows:", etc., and to use instead things like "as shown in the following example:". I see many instances of "For example:" throughout this doc, so I assume it's a global consideration. Feel free to do a mass search and see what you find, and can either address each one as needed or consider as a separate task another time. In this case here, would suggest "... Runtime URL, as shown in the following example:" Same for the next one down too. But up to you how to handle at the moment.

Reply:
Done. 
I have changed the instances of "For example:" in the current section for time being. I did the mass search on an entire doc and I can see that there are 23 occurrences. I will change them in a separate task. Thank you for this.